src/components/atoms/render-list/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+
+export function paragraphNumber(
+  entryIndex: number,
+  paragraphIndex: number
+): string {
+  return `${entryIndex + 1}.${paragraphIndex + 1}. `;
+}
+type RenderListProps = {
+  entry: any;
+  paragraph: any;
+  entryIndex: number;
+  paragraphIndex: number;
+};
+
+export function RenderList({
+  entry,
+  paragraph,
+  entryIndex,
+  paragraphIndex,
+}: RenderListProps) {
+  if (paragraph.type === "paragraph") {
+    return paragraph.list.map((option: any, optionIndex: number) => (
+      <p
+        key={`${entry.key}-${paragraphIndex}-${optionIndex}`}
+        className="text-base"
+      >
+        {optionIndex === 0 && paragraphNumber(entryIndex, paragraphIndex)}
+        {option}
+      </p>
+    ));
+  } else if (paragraph.type === "list") {
```

Comment:
**Change**: honestly the job of a component is render even a null value so add a return statement to this component and was easier create two small components.